superduperdb/base/datalayer.py

```diff
@@ -1116,6 +1030,56 @@ def select_nearest(
         logging.info(str(outs))
         return vi.get_nearest(like, db=self, ids=ids, n=n, outputs=outs)
 
+    # TODO deprecate in favour of remote model calls
+    def predict(
```

Comment:
this is same as 
model.predict_in_db ?

Reply:
`predict_one`.

Reply:
Yes, will be deprecated, due to `GraphModel`.